packages/syft/src/syft/client/client.py

```diff
@@ -206,11 +220,19 @@ def _make_post(
 
         return response.content
 
+    def stream_data(self, credentials: SyftSigningKey) -> Response:
+        url = self.url.with_path(self.routes.STREAM.value)
+        response = self.session.get(
+            str(url), verify=verify_tls(), proxies={}, stream=True
+        )
+        return response
+
     def get_node_metadata(
         self, credentials: SyftSigningKey
     ) -> NodeMetadataJSON | SyftError:
         if self.proxy_target_uid:
             response = forward_message_to_proxy(
+                node_connection=self,
```

Comment:
remove this and other references where we're passing node connection to forward_message_to_proxy function